controller/sharding/labels.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2023 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sharding
+
+// LabelKubeVelaShardID label key for identify the shard id
+const LabelKubeVelaShardID = "core.oam.dev/shard-id"
```

Comment:
Do we need to manage all the label & anno consts in a specific package?

Reply:
I think it is not true for all the time. Whether to manage label in the specific package depends on the scope it needs to be used.

For example, the ShardID might not only be used in KubeVela Application but could potentially be reused in WorkflowRun in the future if we have intense usage for WorkflowRun and need to handle performance issues. So make the label here is more helpful for reuse it across different repos.